composer.json

```diff
@@ -18,18 +18,24 @@
   "require": {
     "php": "^8.1",
     "typo3/cms-form": "^11.5 || ^12.3",
-    "typo3/cms-core": "^11.5 || ^12.3",
     "typo3/cms-frontend": "^11.5 || ^12.3",
     "typo3/cms-install": "^11.5 || ^12.3",
-    "typo3/cms-filelist": "^11.5 || ^12.3"
+    "typo3/cms-filelist": "^11.5 || ^12.3",
+    "typo3/cms-core": "^12.4"
```

Comment:
Now you are using typo3/cms-core ^12 only in composer json. That should be 11 and 12 right?